Source/santad/ProcessTree/process_tree.proto

```diff
@@ -3,4 +3,11 @@ syntax = "proto3";
 package santa.pb.v1.process_tree;
 
 message Annotations {
+  enum Originator {
+    UNSPECIFIED = 0;
+    LOGIN = 1;
+    CRON = 2;
+  }
+
+  Originator ancestor = 1;
```

Comment:
Was there a particular reason to not name this `originator`? Given `ancestor` is more generic, could potentially step on toes of future annotators.

Reply:
Nope, just an oversight from earlier revisions.